### ISSUE TITLE
Does not remove all no longer needed files in %ProgramFiles(x86)%\ folder

### ISSUE THREAD

**Issue by qaqz111** [Microsoft/VisualStudioUninstaller#59]
I uninstalled VS2015 and used VisualStudioUninstaller to clean up, but the folder "**Microsoft Visual Studio 14.0**" was leaved in **%ProgramFiles(x86)%\\**.

There are many empty nested folders here, and extensions are leaved in this place which occupied a lot of disk space. Actually they are safe to be deleted - if the user is not going to reinstall and use VS any more.

So, maybe there should be a **command line option** to force delete those no longer needed files?

**Comment by citron** [comment]
Hello 
I do confirm that the uninstaller does not remove the **"C:\Program Files (x86)\Microsoft Visual Studio 14.0"** folder. 
Can I really safely manualy delete it?

**Comment by qaqz111** [comment]
Yes it is safe to delete this folder after running this uninstaller.

In fact there are still many files/folders/registries left over can be deleted safely, you can use search tools like Everything to locate and delete them.

**Comment by Rabadash8820** [comment]
I also confirm these folders are not removed.  These seem like the most "obvious" artifacts left behind after a VS uninstall; I wouldn't have even thought to look for stale keys in the Registry, but in Program Files?  I can't believe that a little tool requiring an elevated command prompt _still_ leaves these folders behind.
